vocabularies/registryroles.ttl

```diff
@@ -53,23 +53,19 @@ iso11179:ExecutiveCommittee a skos:Concept ;
     skos:inScheme <http://linked.data.gov.au/def/iso11179-6/RolesAndResponsibilities> ;
     skos:prefLabel "Overall Registration Authority"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/iso11179-6/RolesAndResponsibilities> .
 
 iso11179:OverallResponsibleOrganization a skos:Concept ;
-    skos:definition """Responsible Organizations are usually designated by an organizational unit to insure consistence of related Administered Items managed by its Submitting Organizations. In the absence of a designated Responsible Organization, a Submitting Organization should act as a Responsible Organization.\n
-A Responsible Organization is the organization, or part thereof, that is responsible for the integrity and accuracy of the attributes values of the Administered Item; e.g., the semantics of Administered Items maintained and controlled by a Registration Authority. The Responsible Organization is the subject matter expert for the Administered Item.\n
-The Responsible Organization, at the Registration Authority&#39;s request, should review proposals from Submitting Organizations on relevant attributes, e.g., name, definition, and permissible values for the Administered Item&#39;s attributes. The Responsible Organization should inform the Registration Authority of any essential modifications in the specification of the assigned Administered Items."""@en ;
+    skos:definition """Responsible Organizations are usually designated by an organizational unit to ensure consistency of related Administered Items managed by its Submitting Organizations. In the absence of a designated Responsible Organization, a Submitting Organization should act as a Responsible Organization. A Responsible Organization is: The organization, or part thereof, that is responsible for the integrity and accuracy of the attributes values of the Administered Item, e.g., the semantics of Administered Items maintained and controlled by a Registration Authority; the subject-matter expert for the Administered Item; the Organization, at the Registration Authority&#39;s request, that should review proposals from Submitting Organizations on relevant attributes, e.g., name, definition, and permissible values for the Administered Item&#39;s attributes; and the Organization that should inform the Registration Authority of any essential modifications in the specification of the assigned Administered Items."""@en ;
```

Comment:
```suggestion
    skos:definition """Responsible Organizations are usually designated by an organizational unit to ensure consistency of related Administered Items managed by its Submitting Organizations. In the absence of a designated Responsible Organization, a Submitting Organization should act as a Responsible Organization. A Responsible Organization is: The organization, or part thereof, that is responsible for the integrity and accuracy of the attributes values of the Administered Item, e.g., the semantics of Administered Items maintained and controlled by a Registration Authority; the subject-matter expert for the Administered Item; the Organization, at the request of the Registration Authority, that should review proposals from Submitting Organizations on relevant attributes, e.g., name, definition, and permissible values for the attributes of the Administered Item; and the Organization that should inform the Registration Authority of any essential modifications in the specification of the assigned Administered Items."""@en ;
```

Reply:
Thanks Vance for the explanation. Now looks OK for merging.